libparsec/src/config.rs

```diff
@@ -31,3 +31,53 @@ impl From<ClientConfig> for libparsec_client::ClientConfig {
         }
     }
 }
+
+pub fn get_default_data_base_dir() -> PathBuf {
+    #[cfg(target_arch = "wasm32")]
+    {
+        PathBuf::from("")
+    }
+    #[cfg(not(target_arch = "wasm32"))]
+    {
+        // TODO: Windows path differs from Parsec v2 here:
+        // - previous: `%APPDATA%\parsec\data`
+        // - current: `%APPDATA%\Roaming\parsec-v3-alpha`
```

Comment:
Same here, `%APPDATA%` already points to `\Roaming`

Reply:
```suggestion
        // - current: `%APPDATA%\parsec-v3-alpha`
```

Reply:
I based this comment on Parsec v2.16 code: https://github.com/Scille/parsec-cloud/blob/75da5c338ba837d25e09a75b732dbc59de44b551/parsec/core/config.py#L23-L44

Have you experienced the config/data folder to be in another directory with Parsec v2 ?

Reply:
[`dirs::data_dir`](https://docs.rs/dirs/5.0.1/dirs/fn.data_dir.html) correspond to `<user>\AppData\Roaming` (i.e. `%APPDATA%`)